Command/SystemInfoDumpCommand.php

```diff
@@ -8,25 +8,55 @@
  */
 namespace EzSystems\EzSupportToolsBundle\Command;
 
+use EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector;
```

Comment:
Ok, but that should go into a followup methinks.

"Gatherer" or "Hoarder" should be safe :) http://www.thesaurus.com/browse/collect?s=t